pkgs/development/libraries/libpanel/default.nix

```diff
@@ -41,18 +52,23 @@ stdenv.mkDerivation rec {
   ];
 
   mesonFlags = [
-    "-Dinstall-examples=true"
+    (lib.mesonBool "install-examples" true)
+    (lib.mesonEnable "docs" true)
```

Comment:
This is already [`auto` by default](https://gitlab.gnome.org/GNOME/libpanel/-/blob/1.6.0/meson_options.txt#L16-18) and we have `auto_features=enabled`.